sprite.go

```diff
@@ -1759,3 +1766,105 @@ func (pself *SpriteImpl) DeltaTime() float64 {
 func (pself *SpriteImpl) TimeSinceLevelLoad() float64 {
 	return time.TimeSinceLevelLoad()
 }
+
+// ------------------------ sound ----------------------------------------
+
+// Play func:
+//
+//	Play(sound)
+//	Play(video) -- maybe
+//	Play(media, wait) -- sync
+//	Play(media, opts)
+func (p *SpriteImpl) Play__0(media Sound) {
```

Comment:
In Scratch & spx v1, sound-related APIs are provided by `Game` instead of `Sprite`.